priv/repo/migrations/20180413164621_add_slug_to_announcements.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Constable.Repo.Migrations.AddSlugToAnnouncements do
+  use Ecto.Migration
+
+  def change do
+    alter table(:announcements) do
+      add :slug, :string
```

Comment:
What do you think about writing a functino to backfill these in this migration and then changing this to `null: false?

Reply:
I do like that idea. I know that in Rails when I'm going to do something like that I drop straight into sql so that the migration does not depend on whether or not the code modules exist (if someone runs this later and things have changed). But here I might want to use the `Slugger.slugify_downcase/1` function slugify the titles. 

Do you think that would be a bad idea?

Reply:
I think given how simple it is, I'd just copy the code over. If that code evolves over time, it wouldn't hurt the migrations but if the original module disappeared, it makes life harder.

Reply:
Okay. That sounds good. I'll move the code that's in the mix task in here then, and hope that no one runs into issues. I hope we don't get rid of the `Announcements` module in an application about announcements. 😁

Reply:
The work is in a different branch, since I'm going with a different approach, but I addressed this comment in that branch (See https://github.com/thoughtbot/constable/commit/5031f773d20ee23cc243a178ad3a7b6604a72950).